File: src/0026.py
class Solution:
    def removeDuplicates(self, nums: list[int]) -> int:
        temp = 101
        p = 0
        for n in nums:
            if temp == n:
                continue
            else:
                nums[p] = n
                p += 1
            temp = n
        return p

File: src/0026.cpp
#include <vector>

using namespace std;

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int temp = 101, p = 0;
        for(int i = 0; i < nums.size(); ++i) {
            if(temp == nums[i]) {
                continue;
            }else {
                nums[p] = nums[i];
                ++p;
            }
            temp = nums[i];
        }
        return p;
    }
};
